### PR TITLE
Feature/code samples length

### PR DESCRIPTION
**Pull request by karellm**
As discussed in #289, this PR avoid horizontal scrolls in code example in the website. Since it is the first addon I contributed to and it took me couple minutes to figure out how to run the website, I added a `contributing.hbs` page too.

```hbs
# bad :(
{{#power-select options=countries searchField="name" selected=selected onchange=(action (mut selected)) as |country|}}
  {{country.name}}
{{/power-select}}

# good :)
{{#power-select
  options=countries
  searchField="name"
  selected=selected
  onchange=(action (mut selected))
  as |country|
}}
  {{country.name}}
{{/power-select}}
```

### CODE REVIEW

File: tests/dummy/app/templates/public-pages/docs/contributing.hbs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ember app. For instance you can find this page in the
+  <code>tests/dummy/public-pages/docs/contributing.hbs</code> template. You
+  can run the website locally:
+</p>
+
+<pre>
+  <code>
+    npm install
+    bower install
+    cd tests/dummy
```
Comment (cibernox) [inline tests/dummy/app/templates/public-pages/docs/contributing.hbs]: Actually you don't need to cd into tests/dummy. Just `ember s` is enough.

**Comment by cibernox** [comment]
Just as a personal taste, I prefer 

```hbs
{{#power-select
  options=countries
  searchField="name"
  selected=selected
  onchange=(action (mut selected))
  as |country|}}
  {{country.name}}
{{/power-select}}
```
over 
```hbs
{{#power-select
  options=countries
  searchField="name"
  selected=selected
  onchange=(action (mut selected))
  as |country|
}}
  {{country.name}}
{{/power-select}}
```

**Comment by karellm** [comment]
That's why I asked in #289 if you were fine with the style ;) Is that really a deal breaker? 

The reason I think it is better to close on a new line is that you can change options order (think of components that don't finish with `as |foo|`. You can also remove the last line if you need to without having to care for the closing part. And last but not least, the fact that `}}` is on a new line give you a visual clue of where the block start. I think indentation based language are terrible (but this on is a matter of taste).

**Comment by cibernox** [comment]
That is the style I follow in the code of the component and probably should be consistent. I never considered reordering the options. It just looks ugly to me to see the closing braces there.

Also, about the contributing, I'm perhaps more used to see it in the `README.md` of the project than in the page. I think that this belongs more to github's readme than to the documentation.

Tests are failing for so syntax error somewhere.

File: tests/dummy/app/templates/public-pages/docs/contributing.hbs (hunk 1 above)
Comment (karellm) [inline tests/dummy/app/templates/public-pages/docs/contributing.hbs]: :+1:

Comment (cibernox) [inline tests/dummy/app/templates/public-pages/docs/contributing.hbs]: Check the other comment. I think this explanations fit better in the readme of the project than in the web.

**Comment by karellm** [comment]
@cibernox I had couple typos indeed. I amended the changes and made a CONTRIBUTING.md document with a reference from the README. Let me know if there is anything else.

**Comment by cibernox** [comment]
Looks perfect to me now. Let's wait for CI to finish and I'll merge it.

**Comment by karellm** [comment]
@cibernox I just amended a change, the Contributing was referencing the fact it was in the docs.

**Comment by cibernox** [comment]
Thanks! I will deploy soon.

**Pull request merged**